dashboard-ui/src/components/ActionsCompare/decisionMakerDetails.jsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import DecisionList from "./decisionList";
+import DecisionMakerDash from "./decisionMakerDash";
+import { Tabs, Tab, Dropdown, Row, Col } from 'react-bootstrap';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+
+const getPerformerADMByScenarioName = "getPerformerADMsForScenario";
+const performer_adm_by_scenario = gql`
+    query getPerformerADMsForScenario($scenarioID: ID){
+        getPerformerADMsForScenario(scenarioID: $scenarioID)
+    }`;
+
+class DecisionMakerDetails extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            dropdownLabel: "Choose Decision Maker",
+            decisionMaker: "",
+            isHuman: false
+        };
+    }
+
+    handleDecisionMakerSelect = (dropdownLabel) => {
+        // Check if the selected choice contains the word "Human"
+        const isHuman = dropdownLabel.toLowerCase().includes("human");
+        let decisionMakerChoice = ""
+        if (isHuman) {
+            decisionMakerChoice = dropdownLabel === "Human 1" ? "human1" : "human2"
+        } else {
+            decisionMakerChoice = dropdownLabel
+        }
+        this.setState({
+            dropdownLabel: dropdownLabel,
+            decisionMaker: decisionMakerChoice,
+            isHuman: isHuman
+        });
+    };
+
+    componentDidUpdate(prevProps) {
+        if (this.props.selectedScenario !== prevProps.selectedScenario) {
+            this.setState({dropdownLabel: "Choose Decision Maker", decisionMaker: "", isHuman: false})
+        }
+    }
+
+    formatADMString(peformerADMString) {
+        if(peformerADMString.indexOf("ALIGN") > -1 ) {
+            return ("Kitware: " + peformerADMString);
+        } else if (peformerADMString.indexOf("TAD") > -1){
+            return ("Parallax: " + peformerADMString);
+        } else {
+            return peformerADMString;
+        }
+    }
+
+    render = () => {
+        return (
+                <Row>
+                    <Col>
+                        <Tabs className="pt-1 px-1">
+                            <Tab eventKey="0" title="Action List" className="p-4">
+                                <DecisionList title={this.state.dropdownLabel} decisionMaker={this.state.decisionMaker} isHuman={this.state.isHuman} selectedScenario={this.props.selectedScenario}/>
+                            </Tab>
+                            {/* commenting out the scenario stats until I am parse the adm data to match the same format
```

Comment:
Ideally we have a way to get the scenario stats into the database and then use a query to pull them out.